test_data/santa.log

```diff
@@ -192,3 +192,15 @@
 [2018-08-19T04:04:01.479Z] I santad: action=WRITE|path=/private/var/log/powermanagement/StoreData|pid=39|ppid=1|process=syslogd|processpath=/usr/sbin/syslogd|uid=0|user=root|gid=0|group=wheel
 [2018-08-19T04:04:01.480Z] I santad: action=DISKDISAPPEAR|mount=|volume=EFI|bsdname=disk2s1
 [2018-08-19T04:04:01.480Z] I santad: action=DISKDISAPPEAR|mount=|volume=Untitled|bsdname=disk2s2
+[2022-02-09T16:47:01.893Z] I santad: action=EXEC|decision=ALLOW|reason=CERT|sha256=6107671d8e280d9c00f34fdcff9875fbeddddaf5dbf458612383fa20f5ad67b2|cert_sha256=d84db96af8c2e60ac4c851a21ec460f6f84e0235beb17d24a78712b9b021ed57|cert_cn=Software Signing|pid=75780|pidversion=1765713|ppid=155|uid=0|user=root|gid=1|group=daemon|mode=L|path=/bin/ps|args=/bin/ps -x -o utime,stime -p 57680
```

Comment:
since this a different version of the format I opt to move these to test_data/santa2.log or equivalent

Reply:
I moved the new log entries to santa2.log and updated unit tests